frontend_app_workspace_advanced/src/component/SpaceDescription.jsx

```diff
@@ -1,86 +1,95 @@
 import React, { useEffect, useState } from 'react'
 import PropTypes from 'prop-types'
 import {
-  AutoComplete,
   IconButton,
-  tinymceRemove
+  TinyEditor,
+  replaceHTMLElementWithMention,
+  searchContentAndPlaceBalise,
+  searchMentionAndPlaceBalise
 } from 'tracim_frontend_lib'
 import { translate } from 'react-i18next'
 
 export const SpaceDescription = (props) => {
-  const [showEditionMode, setShowEditionMode] = useState(false)
-
-  const initWysiwyg = () => globalThis.wysiwyg(
-    `#${props.textareaId}`,
-    props.lang,
-    props.onChangeDescription,
-    props.onTinyMceInput,
-    props.onTinyMceKeyDown,
-    props.onTinyMceKeyUp,
-    props.onTinyMceSelectionChange
-  )
+  const [content, setContent] = useState('')
+  const [isModeEdition, setIsModeEdition] = useState(false)
 
   useEffect(() => {
-    if (showEditionMode) initWysiwyg()
-  }, [showEditionMode])
+    setContent(props.description)
+  }, [props.description])
 
   useEffect(() => {
-    if (showEditionMode) {
-      tinymceRemove(`#${props.textareaId}`)
-      initWysiwyg()
+    if (isModeEdition) {
+      const contentWithMention = replaceHTMLElementWithMention(
+        props.roleList,
+        props.memberList,
+        content
+      )
+      setContent(contentWithMention)
     }
-  }, [props.lang])
+  }, [isModeEdition])
 
-  useEffect(() => {
-    return () => {
-      if (!props.isReadOnlyMode) tinymceRemove(`#${props.textareaId}`)
-    }
-  }, [props.isReadOnlyMode])
+  /**
+   * Send the description to the backend
+   * @param {string} description The description to send
+   */
+  const sendDescription = async (description) => {
+    const parsedMentionCommentObject = searchMentionAndPlaceBalise(
```

Comment:
balise is a french word, prefer "tag"

also, shouldn't it be named "seachMentionAndReplaceWithTag"?